<ui version="4.0" >
 <class>CaricaGioco</class>
 <widget class="QDialog" name="CaricaGioco" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Carica Gioco</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QTableWidget" name="tableFile" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="newButton" >
         <property name="text" >
          <string>Nuovo...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="caricaButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Carica...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancellaButton" >
         <property name="text" >
          <string>Cancella</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancellaButton</sender>
   <signal>clicked()</signal>
   <receiver>CaricaGioco</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>294</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel" >
     <x>270</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newButton</sender>
   <signal>clicked()</signal>
   <receiver>CaricaGioco</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>90</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel" >
     <x>71</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
